bugbug/db.py

```diff
@@ -132,7 +100,7 @@ def last_modified(path):
 
     if last_modified is None:
         base_url = os.path.splitext(url)[0]
-        last_modified = utils.get_last_modified(f"{base_url}.xz")
+        last_modified = utils.get_last_modified(f"{base_url}.zst")
```

Comment:
You're trying with zst twice here (the first time with the unmodified URL and the second time here). You can instead completely remove this branch